cpp_format_tools/Dockerfile

```diff
@@ -1,16 +1,16 @@
 ARG BUILDIFIER_VERSION=3.5.0
-ARG CLANG_VERSION=10.0.1-r0
+ARG CLANG_VERSION=17.0.5-r0
```

Comment:
If we use clang 17, will the result of clang-format be the same as currently with version 10 ?

Reply:
Resolving, we since decided to upgrade clang-format anyway.